content/docs/tasks/security/mutual-tls/index.md

```diff
@@ -69,11 +69,19 @@ Please check [Istio identity](/docs/concepts/security/#istio-identity) for more
 
 ## Verify mutual TLS configuration
 
-You can use the `istioctl` tool to check the effective mutual TLS settings. To identify the authentication policy and destination rules used for the
-`httpbin.default.svc.cluster.local` configuration and the mode employed, use the following command:
+For this task, you may want to get the pod ID of the client app (`sleep`) with the following command:
```

Comment:
Will readers need that information later? You are even storing the value in and environment variable so my guess is yes. 
What do readers need to change in the command? (my guess is nothing but I have to ask)
I suggest you let readers know what the command is doing and why they need to run it:
```suggestion
This task uses the pod ID of the `sleep` client app. Store it in the `SLEEP_POD` environment variable with the following command:
```

Reply:
I move this command to the related block to keep them closed.